services/submission/src/main/java/app/coronawarn/server/services/submission/validation/PrintableSubmissionPayload.java

```diff
@@ -40,16 +43,20 @@ public String toString() {
         .append(VISITED_COUNTRIES_MESSAGE).append(visitedCountries)
         .append(CONSENT_MESSAGE).append(consentToFederation)
         .append(PADDING_MESSAGE).append(padding.size());
-    stringBuilder.append(" keys: ");
+    if (isEmpty(keys)) {
+      stringBuilder.append(" keys: are empty!");
+    } else {
+      stringBuilder.append(" " + keys.size() + " keys: ");
+    }
     for (TemporaryExposureKey key : keys) {
       stringBuilder
-          .append(" {")
-          .append(" key_data: HIDDEN")
-          .append(" transmission_risk_level: ").append(key.getTransmissionRiskLevel())
-          .append(" rolling_start_interval_number: ").append(key.getRollingStartIntervalNumber())
-          .append(" report_type: ").append(key.getReportType())
-          .append(" days_since_onset_of_symptoms: ").append(key.getDaysSinceOnsetOfSymptoms())
-          .append(" }");
+          .append("{")
+          .append("data: HIDDEN")
+          .append(",trl: ").append(key.getTransmissionRiskLevel())
+          .append(",rsin: ").append(key.getRollingStartIntervalNumber())
+          .append(",type: ").append(key.getReportType())
+          .append(",dsoos: ").append(key.getDaysSinceOnsetOfSymptoms())
+          .append("},");
```

Comment:
I think for the last key a comma will be added as well 🤔

Reply:
true, but it doesn't harm, because it's just for debug/logging purpose and javascript/json is very fault tolerant on dangling commas